### PR TITLE
front: playwright: Adds tests that check if the infrastructure list is displayed correctly in stdcm

### LINKED ISSUE #2972: Check if the application correctly displays the infrastructures list
## Description & Goal

Implement an E2E test that verifies that the infrastructures list is displayed correctly.

## To do

- [ ] Implement axios request interceptor 
- [ ] Fulfill respons with test data
- [ ] Test if every infrastructure is properly displyed 

## Acceptance criteria

- [ ] CI tests must pass  
- [ ] The infrastructures list must be displayed correctly

### PR DESCRIPTION
**Pull request by Uriel-Sautron**
close #2972

### CODE REVIEW

File: front/tests/stdcm-page-model.ts [hunk 1]
```diff
@@ -0,0 +1,55 @@
+/* eslint-disable import/prefer-default-export */
+import { expect, Locator, Page } from '@playwright/test';
+import infraManagement from '../public/locales/fr/infraManagement.json';
+
+export class PlaywrightSTDCMPage {
+  // The current page object
```
Comment (bloussou) [inline front/tests/stdcm-page-model.ts]:
```suggestion
```
I would remove these comments as named of the variables are clear enough

**Review by bloussou — COMMENTED** [review]
It's cool :star: I think most of the comments are paraphrasing the code and therefore could be removed without making the code unreadable

File: front/tests/stdcm-page-model.ts (hunk 1 above)
Comment (Uriel-Sautron) [reply front/tests/stdcm-page-model.ts]: Thanks for the comments. Okay, if it's clear, I'll remove the comments.

Comment (Uriel-Sautron) [reply front/tests/stdcm-page-model.ts]: Done

**Review by Math-R — APPROVED** [review]

**Pull request merged**